frontend/wallet/src/render/DownloadLogic.re

```diff
@@ -109,3 +120,22 @@ let rec download = (filename, url, encoding, maxRedirects, chunkCb, doneCb) => {
     );
   Https.Request.onError(request, e => doneCb(Error(e)));
 };
+
+let codaRepo = "...";
+[@bs.module "electron"] [@bs.scope "app"]
+external getAppPath: string => string = "getPath";
```

Comment:
Can we use the definition from https://github.com/CodaProtocol/coda/blob/develop/frontend/wallet/src/main/ProjectRoot.re